solidity/dashboard/src/services/liquidity-rewards.js

```diff
@@ -422,20 +419,24 @@ class TokenGeyserLPRewards extends LiquidityRewards {
 }
 
 const LiquidityRewardsPoolStrategy = {
-  UNISWAP: UniswapLPRewards,
-  SADDLE: SaddleLPRewards,
-  TOKEN_GEYSER: TokenGeyserLPRewards,
+  [POOL_TYPE.UNISWAP]: UniswapLPRewards,
+  [POOL_TYPE.SADDLE]: SaddleLPRewards,
+  [POOL_TYPE.TOKEN_GEYSER]: TokenGeyserLPRewards,
 }
 
 export class LiquidityRewardsFactory {
   /**
    *
-   * @param {('UNISWAP' | 'SADDLE' | 'TOKEN_GEYSER')} pool - The supported type of pools.
-   * @param {Object} LPRewardsContract - The LPRewardsContract as web3 contract instance.
+   * @param {('UNISWAP' | 'SADDLE' | 'TOKEN_GEYSER')} pool - The supported type
```

Comment:
We have `POOL_TYPE` constant defined, so maybe we could use it here as a `pool` type instead of `{('UNISWAP' | 'SADDLE' | 'TOKEN_GEYSER')}`

Reply:
We can add jsdoc for the `POOL_TYPE` variable like this:
```
/**
 * Enum defines a type of the liquidity pools.
 * @enum {string}
 */
const POOL_TYPE = { ... }
```

and then use it instead of `{('UNISWAP' | 'SADDLE' | 'TOKEN_GEYSER')}`. What do you think?

Reply:
Let's keep it as it is